BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -257,19 +275,25 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
   }
   
   @objc private func tappedRewardsSummaryButton() {
-    let contentView = walletView.contentView
     let rewardsSummaryView = walletView.rewardsSummaryView
     
     let isExpanding = rewardsSummaryView.transform.ty == 0
+    transformRewardSummaryView(isExpanding: isExpanding, animated: true)
+  }
+  
+  private func transformRewardSummaryView(isExpanding: Bool, animated: Bool) {
+    let contentView = walletView.contentView
+    let rewardsSummaryView = walletView.rewardsSummaryView
+    
     rewardsSummaryView.rewardsSummaryButton.slideToggleImageView.image =
       UIImage(frameworkResourceNamed: isExpanding ? "slide-down" : "slide-up")
     
     // Animating the rewards summary with a bit of a bounce
-    UIView.animate(withDuration: 0.4, delay: 0, usingSpringWithDamping: 0.85, initialSpringVelocity: 0, options: [], animations: {
+    UIView.animate(withDuration: animated ? 0.4 : 0.0, delay: 0, usingSpringWithDamping: 0.85, initialSpringVelocity: 0, options: [], animations: {
       if isExpanding {
         rewardsSummaryView.transform = CGAffineTransform(
           translationX: 0,
-          y: -self.walletView.summaryLayoutGuide.layoutFrame.height + rewardsSummaryView.rewardsSummaryButton.bounds.height
+          y: -self.walletView.bounds.height + self.walletView.headerView.bounds.height + 20 + rewardsSummaryView.rewardsSummaryButton.bounds.height
```

Comment:
Is there a reason you're no longer using the summary layout guide?

Reply:
Summary layoutGuide is actually dependent on the preferredContentSize to do the resize. We need a value here before/ during the animation is happening.